### DISCUSSION TITLE
Request for HEK293 GTF/genome/transcriptome files used in paper

### DISCUSSION THREAD

**Discussion by cathoderaymission** [GoekeLab/xpore#81] [Q&A] [answered]
Per issue #73, a request for the full GTF/genome/transcriptome files used for processing the entire HEK293 dataset.

I am currently attempting to process the entire HEK293 dataset using a transcriptome assembled via "gffread" from GRCh38.91 from ensembl (per the xpore paper) along with the same version GTF file and it is not working.

```gffread Homo_sapiens.GRCh38.91.gff3 -W -O -g Homo_sapiens.GRCh38.dna.primary_assembly.fa  -w GRCh38.transcript.fa```

I am getting an empty data.json files when running:

```
xpore dataprep \
--eventalign $DIR/eventalign.txt \
--gtf_path_or_url /path/to/Homo_sapiens.GRCh38.91.gtf \
--transcript_fasta_paths_or_urls /path/to/GRCh38.19.transcript.fa \
--out_dir $DIR/dataprep \
--genome --n_processes 32```

**Comment by ploy-np** [comment, marked as answer]
Hi @cathoderaymission ,

Please see #86.
